### ISSUE TITLE
Tooltip flickering issue introduced in v6.10.4

### ISSUE THREAD

**Issue by alarner** [segmentio/evergreen#1548]
**Describe the bug**
When using the <Tooltip> component on an IconButton the tooltip flickers in the upper left hand corner.

**To Reproduce**

1. Add a tooltip, eg.
```
<Tooltip content="Tooltip 1">
  <IconButton
    icon={BuildIcon}
    intent="none"
    appearance="default"
  />
</Tooltip>
```
2. Hover over the tooltip
4. Observe flickering tooltip in upper left hand corner of the page

Demo Link: https://codesandbox.io/s/priceless-banach-7x7yqo?file=/src/App.js:168-327

**Expected behavior**
Prior to version 6.10.4 this flickering did not occur. I noticed it when upgrading to the newest version (currently v6.13.0) and traced it back to a change that occurred in v6.10.4. The flickering does not exist in v6.10.3.

**Evergreen Version**
v6.10.4

**Environment (please complete the following information):**

- OS: Mac OS X
- Browser: Electron and Firefox
- Version: Electron 17.0.0 and Firefox 107.0

**Comment by brandongregoryscott** [comment]
Good catch, thanks for tracking down the bad version. I'm thinking the changes to the Portal component may be causing this, but I'll dig into it.

**Comment by MasterKali06** [comment]
hey, i like to work on this bug if that's cool and no one working on it!

**Comment by brandongregoryscott** [comment]
> hey, i like to work on this bug if that's cool and no one working on it!

I've been taking a look at it over the weekend but I'm still testing out my solution to make sure it solves this issue and doesn't break SSR, which was the reason the Portal was recently updated (#1512 / #1517).

I'm hoping to get a PR out soon, but feel free to take a stab at it yourself if you'd like!

**Comment by brandongregoryscott** [comment]
This should be resolved [v6.13.1](https://github.com/segmentio/evergreen/releases/tag/v6.13.1)
